client/asset/dcr/dcr.go

```diff
@@ -1585,19 +1834,39 @@ func (dcr *ExchangeWallet) FundOrder(ord *asset.Order) (asset.Coins, []dex.Bytes
 		dcr.log.Errorf("calcBumpRate error: %v", err)
 	}
 
-	coins, redeemScripts, sum, inputsSize, err := dcr.fund(orderEnough(ord.Value, ord.MaxSwapCount, bumpedMaxRate))
-	if err != nil {
-		return nil, nil, fmt.Errorf("error funding order value of %s DCR: %w",
-			amount(ord.Value), err)
-	}
-
+	// If a split is not requested, but is forced, create an extra output from
+	// the split tx to help avoid a forced split in subsequent orders.
+	var extraSplitOutput uint64
 	useSplit := cfg.useSplitTx
 	if customCfg.Split != nil {
 		useSplit = *customCfg.Split
 	}
 
-	// Send a split, if preferred.
-	if useSplit && !ord.Immediate {
+	changeForReserves := useSplit && cfg.unmixedAccount == ""
+	reserves := dcr.reserves()
+	coins, redeemScripts, sum, inputsSize, err := dcr.fund(reserves,
+		orderEnough(ord.Value, ord.MaxSwapCount, bumpedMaxRate, changeForReserves))
+	if err != nil {
+		if !useSplit && reserves > 0 {
```

Comment:
A big part of this forced split is that is does work for mixing wallets. That's why this is `!useSplit` instead of `!changeForReserves`.

Reply:
But the extra output goes to the trading wallet instead of the unmixed wallet.

Reply:
That's fine, the trading account is used to fund stuff and counts toward available balance.  I might be missing your point about this?

Reply:
I don't understand why the forced split doesn't work for mixed wallets. Btw, if someone has a mixed wallet and `useSplit` is true, then the force will not be attempted, but if `useSplit` is false, then the force split will be attempted.

Reply:
> I don't understand why the forced split doesn't work for mixed wallets.

It's certainly supposed to.  Do you mean you're trying it and it's not, or the intent of the code looks to you like it's supposed to block the force split?

> Btw, if someone has a mixed wallet and useSplit is true, then the force will not be attempted, but if useSplit is false, then the force split will be attempted.

That's a good point.  For regular non-mixing wallets that flow makes sense because regular splits have usable change, but not with mixing wallets, so a force split with the extra output should be attempted.

Reply:
Ok, I see now what you mean that when useSplit is true and the wallet is mixed, it can't do the forced split.  Fixing.